tools/ota_e2e_tests/aws_ota_test/aws_ota_test_case_2_updates_cancel_1st.py

```diff
@@ -74,6 +81,9 @@ def run(self):
         if exec_status == 'QUEUED':
             return OtaTestResult(testName=self._name, result=OtaTestResult.ERROR,
                                  summary='Timeout waiting for OTA job status.')
+        if exec_status == 'SUCCEEDED':
+            return OtaTestResult(testName=self._name, result=OtaTestResult.ERROR,
+                                 summary='OTA update complete too fast before we can cancel the job.')
 
         # Force cancel the first job that's in progress, device should pick up the 2nd update and succeed.
         iot_client = boto3.client('iot')
```

Comment:
Comment on line 90, though unlikely to fail, is it possible to check the status of cancel_job_execution?

Reply:
We don't need to actually. Before it's failing because the update is already finished. Now if it fails we just let the exception pop up and fails the tests.